client/src/pages/User/styles.css

```diff
@@ -0,0 +1,18 @@
+.Container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: 30px;
+}
+
+.Dialog-container{
+    height: "50%";
+}
+
+.Progress{
+    color: #48ff00;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+}
```

Comment:
we are going to use `makeStyles` for styling instead of CSS

Reply:
great news :)